PokemonAPI/PokemonAPI.BusinessLayer/Implementations/DomainServices/CatchService.cs

```diff
@@ -33,8 +36,15 @@ public async Task CatchPokemon(Guid pokemonId, Guid trainerId, CancellationToken
             if (pokemonEntity.Trainer != null)
                 throw new CatchPokemonException("Pokemon already has a trainer");
 
-            if (_chanceGenerator.getChance(2) != 0)
+            var chance = _chanceGenerator.getChance(2);
+
+            _logger.LogInformation("Chance to catch a pokemon is 50%");
```

Comment:
This is kinda too useless of information. We already **know** this because that is the implementation. Something like this would be useful perhaps if the chance were to vary from call to call of the chance generator.